.github/workflows/slither.yaml

```
@@ -20,11 +18,17 @@ jobs:
         uses: crytic/slither-action@v0.4.0
         id: slither
         with:
-          node-version: 18.12.0
-          sarif: results.sarif
           fail-on: none
+          slither-args: --checklist --markdown-root ${{ github.server_url }}/${{ github.repository }}/blob/${{ github.sha }}/
 
-      - name: Upload SARIF file
-        uses: github/codeql-action/upload-sarif@v3
+      - name: Create/update checklist as PR comment
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)